src/app.module.ts

```diff
@@ -54,13 +55,19 @@ import { ProviderService } from './service/provider/provider.service';
 import { PrometheusMetricModule } from './common/prometheus';
 import { ClientService } from './service/client/client.service';
 import { AllocatorReportGeneratorJobService } from './jobs/allocator-report-generator-job/allocator-report-generator-job.service';
+import { CidContactService } from './service/cid-contact/cid-contact.service';
+import { IpniAdvertisementFetcherJobService } from './jobs/ipni-advertisement-fetcher-job/ipni-advertisement-fetcher-job.service';
+import { IpniMisreportingCheckerService } from './service/ipni-misreporting-checker/ipni-misreporting-checker.service';
+import { HttpService } from '@nestjs/axios';
+import axiosBetterStacktrace from 'axios-better-stacktrace';
+import axios from 'axios';
 
 @Module({
   imports: [
     ConfigModule.forRoot(),
     ScheduleModule.forRoot(),
-    HttpModule.register({ timeout: 5000 }),
-    CacheModule.register(),
+    HttpModule.register({ timeout: 5000 }), // 5 seconds
+    CacheModule.register({ ttl: 10000 }), // 10 seconds
```

Comment:
What was the default ttl? There is a lot of places were we rely on the default value.

Reply:
Default is 0, meaning cache with no expiration. This changes all default to 10 seconds unless specified otherwise (example: healtchceck endpoint with 5 seconds cache).

https://docs.nestjs.com/techniques/caching
"The default value for ttl is 0, meaning the cache will never expire"

Reply:
There are 2 GET endpoints that were using default ttl in `src/service/datacapstats/datacapstats.service.ts` for fetching client details from datacapstats. While no expiration we had before for this is probably bad, 10 sec is pointless for this endpoint, it should be ~24h at least. Can we tune it there?